hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -54,6 +54,14 @@ tuiContestationPeriod = UnsafeContestationPeriod 10
 spec :: Spec
 spec = do
   context "end-to-end smoke tests" $ do
+    around setupBadHostNodeAndTUI $ do
+      it "can quit before connected" $
+        \TUITest{sendInputEvent, shouldRender, shouldNotRender} -> do
```

Comment:
As you use it only once, i would suggest to use it directly and not through `around`

```suggestion
    it "can quit before connected" $
      setupBadHostNodeAndTUI $ \TUITest{sendInputEvent, shouldRender, shouldNotRender} -> do
```